cnv/cnv-attaching-vm-multiple-networks.adoc

```diff
@@ -0,0 +1,28 @@
+[id="attaching-to-multiple-networks"]
+= Attaching a virtual machine to multiple networks
+include::modules/cnv_document_attributes.adoc[]
+:context: cnv-attaching-multiple-networks
+toc::[]
+
+{ProductName} provides Layer-2 networking capabilities that allow you to connect
+virtual machines to multiple networks. You can import virtual machines with
+existing workloads that depend on access to multiple interfaces. You can also
+configure a PXE network so that you can boot machines over the network.
+
+To get started, a network administrator configures a NetworkAttachmentDefinition
+of type `cnv-bridge`. Then, users can attach Pods, VMIs, and VMs to the
+bridge network. From the {ProductName} web console, you can create a vNIC
+that refers to the bridge network.
+
+include::modules/cnv-networking-glossary.adoc[leveloffset=+1]
+
+include::modules/cnv-connecting-resource-bridge-network.adoc[leveloffset=+1]
+
+[CAUTION]
+====
+In the next section, you must define a vNIC that refers to the bridge
+network by selecting the NetworkAttachmentDefinition object you created in
+the previous section.
+====
+
+include::modules/cnv-vm-create-nic-web.adoc[leveloffset=+1]
```

Comment:
Actual file name is cnv_vm_create_nic_web.adoc (underscores instead of hypens).

Reply:
Hey, that was the file name previously (in 1.4) but for 2.0, we are switching to hyphens instead of underscores where possible. This is what we should have done all along, but we missed that the first time. The file in this PR has hyphens, so it is the correct file name.